File: dataset_reduzido/codenet/Java/Code/s682242794.java
import java.util.Scanner;
public class Main{
    int high;
    int low;
    int sa;
    
    public void solve(){
        Scanner sc = new Scanner(System.in);
        for(int i=0;i<7;i++){
            high=sc.nextInt();
            low=sc.nextInt();
            sa=high-low;
            System.out.println(sa);
        }
    }
    
    public static void main(String[] args){
        Main obj = new Main();
        obj.solve();
    }
}

File: output/Llama-2-100/codenet/Java/Python/s682242794.py
import input

class Main:
    high = 0
    low = 0
    sa = 0
    
    def solve():
        sc = input. Scanner(sys.stdin)
        for i in range(7):
            high = sc.nextInt()
            low = sc.nextInt()
            sa = high - low
            print(sa)

if __name__ == "__main__":
    obj = Main()
    obj.solve()
